package.json

```diff
@@ -8,10 +8,5 @@
   "dependencies": {
     "glightbox": "^2.0.5",
     "normalize.css": "^8.0.1"
-  },
-  "scripts": {
-    "glightbox-setup": "cp './node_modules/glightbox/dist/css/glightbox.css' './node_modules/glightbox/dist/css/_glightbox.scss'",
```

Comment:
Just wanted to ensure: isn't that if this `cp` is removed in both places (shopware script and here), scss won't get compiled?

Reply:
It's imported from Theme's main.css file directly from node_modules folder